packages/analytics/analytics-utilities/package.json

```diff
@@ -31,7 +31,7 @@
     "stylelint": "stylelint --allow-empty-input './src/**/*.{css,scss,sass,less,styl,vue}'",
     "stylelint:fix": "stylelint --allow-empty-input './src/**/*.{css,scss,sass,less,styl,vue}' --fix",
     "typecheck": "vue-tsc -p './tsconfig.build.json' --noEmit",
-    "test:unit": "cross-env FORCE_COLOR=1 vitest run",
+    "test:unit": "cross-env FORCE_COLOR=1 TZ='US/Eastern' vitest run && TZ='UTC' vitest run",
```

Comment:
question: will all unit tests in this whole package need to be run against both timezones? If not? You may want to set the timezone in the test files instead of globally for all unit tests. 

If the package sizes grows and the T logic isn't needed for all tests, this will double the execution time for all.